spec/controllers/reports_controller_spec.rb

```diff
@@ -0,0 +1,40 @@
+# frozen_string_literal: true
+
+require "rails_helper"
+
+describe ReportsController, type: :controller do
+  let(:user) { build_current_user }
```

Comment:
From the rspec run looks like you just need to instantiate a current academic cycle here.